charmap.txt

```diff
@@ -412,6 +412,10 @@ B_DEF_TEAM1 = FD 3A
 B_DEF_TEAM2 = FD 3B
 @ FD 3C - preiously gActiveBattler
 @ FD 3D - preiously gActiveBattler without Illusion Check
+B_ATK_NAME_WITH_PREFIX2 = FD 3E
+B_DEF_NAME_WITH_PREFIX2 = FD 3F
+B_EFF_NAME_WITH_PREFIX2 = FD 40
+B_SCR_ACTIVE_NAME_WITH_PREFIX2 = FD 41
```

Comment:
what's the reason for those?

Reply:
I understand, all good

Reply:
Yes, because of capitalization of the "the"